htdocs/PHP/internal_libraries/languages/page_content_fr.php

```diff
@@ -0,0 +1,195 @@
+
+$translation["pdf_document_with_expiration"] = "
+<p>Ce document a été créé sur WeExpire.org, une plateforme permettant de créer des notes d'urgence qui ne peuvent être lues que par des contacts de confiance après un décès ou une blessure grave.</p>
+<p>Pour lire la note d'urgence suivante, scannez le code QR et entrez ensuite ce code d'accès : <b>{$_SESSION["initialization_vector"]}</b></p>
+<p>Objet de la note : <b>{$_SESSION["subject"]}</b>
+<p>Date de création de la note : <b>".date("d/m/Y H:i:s", strtotime($_SESSION["creation_date_time"]))."</b> (DD/MM/YYYY, UTC+1)</p>
+<p>Date d'expiration de la note : <b>".date("d/m/Y", strtotime($_SESSION["expiration_date"]))."</b> (DD/MM/YYYY, UTC+1)</p>
+<p></p><p></p>
+<p>Espace pour d'autres commentaires et/ou signature :<br>
+___________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________</p>";
+}
+
+// Privacy Policy page
+$translation["privacy_page_title"] = "Vie privée";
+$translation["privacy_page_content"] = "<p>Notre philosophie est de respecter profondément la vie privée des utilisateurs. WeExpire ne collecte pas de données personnelles ou d'informations personnellement identifiables (PII), ce qui peut être vérifié en consultant notre <a href='https://github.com/ciccionamente/WeExpire' target='_blank'>code en sources libres</a>.</p>
+<p>Nous utilisons <a href='https://plausible.io/data-policy' target='_blank'>Plausible Analytics</a> qui est un service d'analyse de site conforme au RGPD, au CCPA et à la législation sur les cookies. Ce service ne collecte aucune donnée personnelle ou information personnellement identifiable (PII), n'utilise pas de cookies et respecte la vie privée des visiteurs du site.</p>
+<p>Nous utilisons des cookies de session persistants de première partie, nécessaires uniquement pour utiliser notre application et pour vous permettre de créer votre note d'urgence.</p>
+<p>Toutes les données sont chiffrées via SSL/TLS lors de leur transmission de nos serveurs à votre navigateur.</p>
+<p><a href='https://www.vhosting.net' target='_blank'>Notre hébergeur</a> stocke les journaux d'accès et de sécurité pendant 30 jours au maximum afin de surveiller et de maintenir la sécurité de ses serveurs. Nous n'avons aucun contrôle sur ces données.</p>
+<p>Nous pouvons mettre à jour cette politique à l'avenir, si nécessaire, pour nous conformer aux réglementations pertinentes et refléter toute nouvelle pratique.</p>
+<p>Dernière mise à jour : 28 juin 2022.</p>";
+
+// Terms of Service page
+$translation["terms_page_title"] = "Conditions d'utilisation";
```

Comment:
In order to be consistent with the other languages and with the footer label, would it be possible to use as a title "Termes et conditions" instead of "Conditions d’utilisation"?